docs/user-guide/rules/list.md

```diff
@@ -0,0 +1,540 @@
+-   [`media-query-list-comma-newline-before`](../../../lib/rules/media-query-list-comma-newline-before/README.md): Require a newline or disallow whitespace before the commas of media query lists.
+-   [`media-query-list-comma-space-after`](../../../lib/rules/media-query-list-comma-space-after/README.md): Require a single space or disallow whitespace after the commas of media query lists (Autofixable).
+-   [`media-query-list-comma-space-before`](../../../lib/rules/media-query-list-comma-space-before/README.md): Require a single space or disallow whitespace before the commas of media query lists (Autofixable).
+
+### At-rule
+
+-   [`at-rule-empty-line-before`](../../../lib/rules/at-rule-empty-line-before/README.md): Require or disallow an empty line before at-rules (Autofixable).
+-   [`at-rule-name-case`](../../../lib/rules/at-rule-name-case/README.md): Specify lowercase or uppercase for at-rules names (Autofixable).
+-   [`at-rule-name-newline-after`](../../../lib/rules/at-rule-name-newline-after/README.md): Require a newline after at-rule names.
+-   [`at-rule-name-space-after`](../../../lib/rules/at-rule-name-space-after/README.md): Require a single space after at-rule names (Autofixable).
+-   [`at-rule-semicolon-newline-after`](../../../lib/rules/at-rule-semicolon-newline-after/README.md): Require a newline after the semicolon of at-rules (Autofixable).
+-   [`at-rule-semicolon-space-before`](../../../lib/rules/at-rule-semicolon-space-before/README.md): Require a single space or disallow whitespace before the semicolons of at-rules.
+
+### Comment
+
+-   [`comment-empty-line-before`](../../../lib/rules/comment-empty-line-before/README.md): Require or disallow an empty line before comments (Autofixable).
+-   [`comment-whitespace-inside`](../../../lib/rules/comment-whitespace-inside/README.md): Require or disallow whitespace on the inside of comment markers (Autofixable).
+
+### General / Sheet
+
+-   [`indentation`](../../../lib/rules/indentation/README.md): Specify indentation (Autofixable).
+-   [`linebreaks`](../../../lib/rules/linebreaks/README.md): Specify unix or windows linebreaks (Autofixable).
+-   [`max-empty-lines`](../../../lib/rules/max-empty-lines/README.md): Limit the number of adjacent empty lines (Autofixable).
+-   [`max-line-length`](../../../lib/rules/max-line-length/README.md): Limit the length of a line.
+-   [`no-eol-whitespace`](../../../lib/rules/no-eol-whitespace/README.md): Disallow end-of-line whitespace (Autofixable).
+-   [`no-missing-end-of-source-newline`](../../../lib/rules/no-missing-end-of-source-newline/README.md): Disallow missing end-of-source newlines (Autofixable).
+-   [`no-empty-first-line`](../../../lib/rules/no-empty-first-line/README.md): Disallow empty first lines (Autofixable).
+-   [`unicode-bom`](../../../lib/rules/unicode-bom/README.md): Require or disallow Unicode BOM.
+
+## As JSON
```

Comment:
> this section is useful

I agree. Here was a comment about it from just [a week ago](https://github.com/stylelint/stylelint/issues/488#issuecomment-572089504).

> but feel hard to maintain, especially when a new rule is added.

It's an improvement on before when two separate docs files need to be updated: the list of rules and the example config. Now they are, at least, both in one file.

I think we keep it for now as we don't often add rules and because it seems like this is still useful for some users.

Reply:
Make sense. 👍 

Ideally, if we could add metadata to each as [ESLint does](https://eslint.org/docs/developer-guide/working-with-rules#rule-basics), this section could be auto-generated from the metadata. But now is not the time to discuss it. 😄

Reply:
I don't see any sense in this section as well.

Reply:
Let's strip it for now then.

We can always dig it out again in the future.